scripts/build_mkl_gpu.sh

```diff
@@ -0,0 +1,58 @@
+export BML_TESTING=${BML_TESTING:=yes}
+export CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE:=Release}
+export EXTRA_CFLAGS=${EXTRA_CFLAGS:=""}
+export EXTRA_LINK_FLAGS=${EXTRA_LINK_FLAGS:=""}
+export CUDA_TOOLKIT_ROOT_DIR=""
+export CMAKE_Fortran_FLAGS="-fiopenmp -fopenmp-targets=spir64 -DINTEL_SDK"
+export CMAKE_C_FLAGS="-O3 -fiopenmp -fopenmp-targets=spir64 -D__STRICT_ANSI__ -DINTEL_SDK"
+#export CMAKE_C_FLAGS="-O3 -fiopenmp -fopenmp-targets=spir64 -D__STRICT_ANSI__ -DINTEL_SDK  -I/soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/mkl/20210313/include"
+#export CMAKE_C_FLAGS="-O3 -fopenmp-targets=spir64 -DINTEL_SDK -D__STRICT_ANSI__ -fiopenmp -DUSE_OMP_OFFLOAD -Wno-missing-prototype-for-cc"
+
+#export CMAKE_C_FLAGS="-O3 -DNDEBUG -fiopenmp -fopenmp-targets=spir64 -fsycl -fsycl-device-code-split=per_kernel /soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/mkl/20210313/lib/intel64/libmkl_sycl.so /soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/mkl/20210313/lib/intel64/libmkl_intel_ilp64.so /soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/mkl/20210313/lib/intel64/libmkl_intel_thread.so /soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/mkl/20210313/lib/intel64/libmkl_core.so -lstdc++ -Wl,-rpath,/soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/kokkos/20210323-3.1/../../compiler/latest/linux/compiler/lib/intel64 -L/soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/kokkos/20210323-3.1/../../compiler/latest/linux/compiler/lib/intel64 -liomp5 -lsycl -lOpenCL -lm -lpthread -ldl"
+
+#works 
+#export EXTRA_LINK_FLAGS="-L/soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/mkl/2021u3_20210525/lib/intel64 -lmkl_lapack95_lp64 -lmkl_intel_lp64 -lmkl_core -lmkl_intel_thread -lpthread -lm -lmkl_intel_ilp64  -lmkl_sycl -lstdc++" 
+export EXTRA_LINK_FLAGS="-L/soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/mkl/2021u3_20210525/lib/intel64 -lmkl_lapack95_lp64 -lmkl_intel_lp64 -lmkl_core -lmkl_intel_thread -lpthread -lm -lmkl_intel_ilp64  -lmkl_sycl -lstdc++ -Wl,-rpath,/soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/kokkos/20210323-3.1/../../compiler/latest/linux/compiler/lib/intel64 -L/soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/kokkos/20210323-3.1/../../compiler/latest/linux/compiler/lib/intel64 -liomp5 -lsycl -lOpenCL -lm -lpthread -ldl"
+
+#export EXTRA_LINK_FLAGS="-fsycl -fsycl-device-code-split=per_kernel /soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/mkl/20210313/lib/intel64/libmkl_sycl.so /soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/mkl/20210313/lib/intel64/libmkl_intel_ilp64.so /soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/mkl/20210313/lib/intel64/libmkl_intel_thread.so /soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/mkl/20210313/lib/intel64/libmkl_core.so -lstdc++ -Wl,-rpath,/soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/kokkos/20210323-3.1/../../compiler/latest/linux/compiler/lib/intel64 -L/soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/kokkos/20210323-3.1/../../compiler/latest/linux/compiler/lib/intel64 -liomp5 -lsycl -lOpenCL -lm -lpthread -ldl"
```

Comment:
There are a lot of commented options in this file. Are they really useful or should they be cleaned up for clarity?

Reply:
it's useless, I forgot to remove it, will clean it. Thanks.